logging/cluster-logging-loki.adoc

```diff
@@ -21,8 +21,12 @@ include::modules/logging-creating-new-group-cluster-admin-user-role.adoc[levelof
 
 include::modules/logging-loki-gui-install.adoc[leveloffset=+1]
 
+include::modules/logging-clo-gui-install.adoc[leveloffset=+1]
```

Comment:
In the Previews comment box, I don't see these `clo` modules, and when I look in the preview itself I can't find them. Can you add links to them?